druid-powered.md

```diff
@@ -34,7 +34,13 @@ Druid powers slice and dice analytics on both historical and realtime-time metri
 
 ## Alibaba
 
-At Alibaba Search Group, we use Druid for real-time analytics of users' interaction with its popular e-commerce site.
+Druid is widely used in different business units of Alibaba Group.
+
+- At Alibaba Search Group, Druid is used to power real-time analytics of users' interaction with its popular e-commerce site.
+
+- At Alibaba Cloud, [Druid](https://www.alibabacloud.com/help/doc-detail/72987.htm) powers its [E-MapReduce](https://www.alibabacloud.com/help/doc-detail/201882.htm) cloud service, which is a big data platform based on open source engines such as Hadoop.
```

Comment:
This is a bit confusing since the Alibaba cloud web page is linked to "Druid" even though the linked page contains explanation of Apache Druid. I would suggest

```suggestion
- At Alibaba Cloud, Druid powers its [E-MapReduce](https://www.alibabacloud.com/help/doc-detail/201882.html) cloud service, which is a big data platform based on open source engines such as Hadoop. See [E-MapReduce Druid](https://www.alibabacloud.com/help/doc-detail/72987.html) for more details.
```